src/test/java/org/opensearch/security/dlic/rest/api/ActionGroupsApiTest.java

```diff
@@ -385,5 +386,6 @@ public void checkNullElementsInArray() throws Exception{
         String body = FileHelper.loadFile("restapi/actiongroup_null_array_element.json");
         HttpResponse response = rh.executePutRequest(ENDPOINT + "/CRUD_UT", body, new Header[0]);
         Assert.assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
+        Assert.assertTrue(response.getBody().contains((ErrorType.NULL_ARRAY_ELEMENT).getMessage()));
```

Comment:
nit: use static import (`ErrorType.NULL_ARRAY_ELEMENT`) and remove unnecessary parentheses.

Reply:
Also it will be better to use the same validation pattern as other error codes. See for example line 155:
```
        Assert.assertEquals(AbstractConfigurationValidator.ErrorType.BODY_NOT_PARSEABLE.getMessage(), settings.get("reason"));
```
As other places do not use static import, it will be OK to use `AbstractConfigurationValidator.ErrorType. NULL_ARRAY_ELEMENT`, or change other occurrences of `AbstractConfigurationValidator.ErrorType.BODY_NOT_PARSEABLE` to `ErrorType.BODY_NOT_PARSEABLE` (the same for other error codes) or change other occurrences to use static import (preferable).